### ISSUE TITLE
Add support for touch events to violent theremin demo

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/content#1448]
@normanr commented on [Wed Dec 02 2020](https://github.com/mdn/sprints/issues/3937)

It would be nice to be able to use the demo in a touch environment (eg: phone or tablet). (Not required: sound could be continuous after a 'start/down' event until an 'end/up' event is received)

---

@chrisdavidmills commented on [Wed Dec 02 2020](https://github.com/mdn/sprints/issues/3937#issuecomment-737085823)

I agree that this would be nice.

For reference:

* Demo repo: https://github.com/mdn/violent-theremin
* Live demo: https://mdn.github.io/violent-theremin/

---

@normanr commented on [Wed Dec 02 2020](https://github.com/mdn/sprints/issues/3937#issuecomment-737473301)

(side-note: the event listeners for triggering `init` should be removed in `init`, otherwise using the keyboard causes init to be triggered multi times, and `DOMExceptions` when trying to remove the `startMessage` that has already been removed).

**Comment by Rumyra** [comment]
Would it be ok to take this on and give it a bit of a re-write (inc touch events) and also move it into the webaudio-examples repo @chrisdavidmills ?

**Comment by chrisdavidmills** [comment]
@Rumyra yes, I'm fine with that. It is mentioned a few times on MDN, so make sure there is some kind of redirect in place.

**Comment by Rumyra** [comment]
Thank you! Yep I think I'm going to roll it into a bigger task of auditing audio, so I'm hoping that won't be too much of a problem 👍
